### PR TITLE
app-admin/abrt: bump package

### PR DESCRIPTION
**Pull request by flappyports**
This includes multiple commits that address dependency requirements and other package bumps which are needed.

### CODE REVIEW

**Comment by flappyports** [comment]
Issues left:
1. strip RHEL/Fedora/CentOS workflows as we do not need them (simple patch)
- See: https://github.com/Sabayon/for-gentoo/tree/master/dev-libs/libreport/files

2. systemd support in various packages?

3. pythondocs? (sphinx-build-3 requirement... is this a naming issue from the Fedora/RHEL world and a simple patch will fix it?)

4. selinux? probably not, but worth considering.

5. Is bodhi required (it breaks the build right now)?

6. Does Gnome team want to enable the bugzilla interface and redirect it to Gentoo?
- See: https://github.com/Sabayon/for-gentoo/tree/master/dev-libs/libreport/files

File: dev-libs/satyr/satyr-0.26.ebuild [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+IUSE=""
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="${PYTHON_DEPS}
+	>=dev-libs/elfutils-0.158
+"
+DEPEND="${RDEPEND}
+	dev-python/sphinx
```
Comment (EvaSDK) [inline dev-libs/satyr/satyr-0.26.ebuild]: Looks like this is not used if not building python2 bindings.

File: dev-libs/satyr/satyr-0.26.ebuild [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+PYTHON_COMPAT=( python3_{4,5,6} )
+
+inherit autotools ltprune python-r1 toolchain-funcs
+
+DESCRIPTION="Satyr is a collection of low-level algorithms for program failure processing"
+HOMEPAGE="https://github.com/abrt/satyr"
+SRC_URI="https://github.com/abrt/${PN}/archive/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
```
Comment (EvaSDK) [inline dev-libs/satyr/satyr-0.26.ebuild]: Please specify the subslot (3).

File: dev-libs/satyr/satyr-0.26.ebuild [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+	./gen-version || die # Needs to be run before full autoreconf
+	eautoreconf
+	python_copy_sources
+}
+
+src_configure() {
+	local myargs=(
+		--localstatedir="${EPREFIX}/var" \
+		# Build breaks without and we aren't supporting Python2 anyway
+		--without-python2
```
Comment (EvaSDK) [inline dev-libs/satyr/satyr-0.26.ebuild]: rpm is either missing from `DEPEND` or `--without-rpm` is needed. Afaict, build succeeds with `--without-rpm`

File: dev-libs/satyr/satyr-0.26.ebuild [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+src_prepare() {
+	default
+	./gen-version || die # Needs to be run before full autoreconf
+	eautoreconf
+	python_copy_sources
+}
+
+src_configure() {
+	local myargs=(
+		--localstatedir="${EPREFIX}/var" \
```
Comment (EvaSDK) [inline dev-libs/satyr/satyr-0.26.ebuild]: The `\` here is superfluous.

File: dev-libs/satyr/satyr-0.26.ebuild (hunk 2 above)
Comment (flappyports) [reply dev-libs/satyr/satyr-0.26.ebuild]: fixed

File: dev-libs/satyr/satyr-0.26.ebuild (hunk 3 above)
Comment (flappyports) [reply dev-libs/satyr/satyr-0.26.ebuild]: fixed

File: dev-libs/satyr/satyr-0.26.ebuild (hunk 4 above)
Comment (flappyports) [reply dev-libs/satyr/satyr-0.26.ebuild]: fixed

File: dev-libs/libreport/libreport-2.9.5.ebuild [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="+gtk"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+COMMON_DEPEND="${PYTHON_DEPS}
+	app-admin/augeas
+	>=dev-libs/glib-2.21:2
```
Comment (EvaSDK) [inline dev-libs/libreport/libreport-2.9.5.ebuild]: configure says 2.43 is the minimum.

Comment (EvaSDK) [reply dev-libs/libreport/libreport-2.9.5.ebuild]: Also it seems there is a dependency on systemd for journal support.

Comment (flappyports) [reply dev-libs/libreport/libreport-2.9.5.ebuild]: fixed dev-libs/glib depend.

will explore the systemd depend later.

File: app-admin/abrt/abrt-2.10.9.ebuild [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+	newconfd "${FILESDIR}/${PN}-2.0.12-r1-conf" abrt # probably needs updated
+}
+
+pkg_preinst() {
+	gnome2_icon_savelist
+}
+
+pkg_postinst() {
+	gnome2_icon_cache_update
+	elog "To start the bug detection service on an openrc-based system, do"
```
Comment (Zlogene) [inline app-admin/abrt/abrt-2.10.9.ebuild]: might be not needed as obvious

File: app-admin/abrt/abrt-2.10.9.ebuild [hunk 7]
```diff
@@ -0,0 +1,116 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5,6} )
+
+# Need gnome2-utils for gnome2_icon_cache_update
+inherit autotools eutils gnome2-utils python-single-r1 systemd user
```
Comment (Zlogene) [inline app-admin/abrt/abrt-2.10.9.ebuild]: seems eutils is not needed

File: app-admin/abrt/abrt-2.10.9.ebuild (hunk 6 above)
Comment (flappyports) [reply app-admin/abrt/abrt-2.10.9.ebuild]: Agreed.  Removed.

File: app-admin/abrt/abrt-2.10.9.ebuild (hunk 7 above)
Comment (flappyports) [reply app-admin/abrt/abrt-2.10.9.ebuild]: Fixed.

File: app-admin/abrt/abrt-2.10.9.ebuild [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+COMMON_DEPEND="${PYTHON_DEPS}
+	>=dev-libs/btparser-0.18
+	>=dev-libs/glib-2.43:2
+	>=dev-libs/libreport-2.9.5[python]
+	dev-libs/libxml2
+	dev-libs/nss
+	>=gnome-base/gsettings-desktop-schemas-3.15.1
+	python? ( ${PYTHON_DEPS} )
```
Comment (EvaSDK) [inline app-admin/abrt/abrt-2.10.9.ebuild]: Python support is actually mandatory (and only py3 btw) as this is needed for cli-ng abrt script.

**Comment by flappyports** [comment]
Thanks @EvaSDK !

**Pull request closed without merging**